src/test/java/com/commercetools/sync/services/impl/ProductServiceTest.java

```diff
@@ -138,33 +136,4 @@ void updateProduct_WithMockCtpResponse_ShouldReturnMock() {
     verify(productSyncOptions.getCtpClient())
         .execute(eq(ProductUpdateCommand.of(productProjectionMock, updateActions)));
   }
-
-  @Test
-  void buildProductKeysQueryPredicate_WithEmptyProductKeysSet_ShouldBuildCorrectQuery() {
```

Comment:
Not sure but, when we refactor the code, is not it better to refactor test instead of dropping them, is there an equivalent of that those queries are correct now ?

Reply:
This tests are completely for the deleted code and yes, for the method I have already added tests in my previous PR.
Check the description.